discord_bot2.py

```diff
@@ -180,9 +278,16 @@ async def on_message(self, message):
         if message.content.startswith('//'):
             return
 
+        if message.channel.name == 'control-duck':
+            await self.control_on_message(message)
+            return
+
```

Comment:
I want something like 
```python
if message.channel.id in self.control_channels:
```
And then we configure the control channels in a configuration file. 

I want to use `id` instead of `name` so we can specify the exact channels on the exact servers. This way admins of other course servers can't simply add a channel named `control-duck` and take over.

Reply:
Something like this?
config.json:
{
  "control_channels": [1111389107587928154]
}